spec/system/user_view_call_details_spec.rb

```diff
@@ -132,6 +132,27 @@
         expect(page).to have_content timer.user.first_plus_initial
       end
     end
+
+    it "displays the time remaining" do
+      freeze_time do
+        unix_time = (Time.now.to_f * 1000).to_i
+        timer = create(:timer, duration: 15.minutes)
+        user = create(:user, group_ids: [timer.call.group_id])
+
+        visit call_path(timer.call, as: user)
+        page.execute_script("mockTime(#{unix_time})")
+
+        within(".timer-details") do
+          expect(page).to have_content "15m 0s"
```

Comment:
I think this test will fail if we decide we want to change the timer format at some point. For example, `15:00` instead of `15m 0s`. Is there a way we could test the timer value without assuming the format displayed on the page?

Reply:
Sort of, yeah. We'll always be testing for specific content because the test is "Does the timer display something the way we expect it to?" One way we could make this test less brittle is by putting this bit of text into the locale file and referencing that:
```yml
en:
  timer:
    minutes_seconds: %{minutes}m %{seconds}s
```
Then we can call this in the ruby/this test like:
```ruby
expect(page)
  .to have_content(t("timer.minutes_seconds", minutes: 15, seconds: 0))
```
And with the help of [this gem](https://github.com/fnando/i18n-js), in the javascript:
```js
I18n.t("timer.minutes_seconds", {minutes: 15, seconds: 0});
```

Doing it this way lets us change the format of the text in a single place.

Reply:
Here's a Go Rails video about it: [Fixing Brittle Tests: Avoid Hardcoded Values](https://gorails.com/episodes/brittle-tests-hardcoded-values)
And a thoughtbot blog post: [Brittle Tests](https://thoughtbot.com/blog/brittle-tests)

Reply:
I'll make a follow up PR adding that gem and getting the locale file and gem set up.